### PR TITLE
responsive styles

### PR DESCRIPTION
**Pull request by paulirish**
image

The css changes are pretty minimal.

However I did also remove the word "Viewer" from the title of everything.
Honestly I think its redundant and it's bugged me for a while.

### CODE REVIEW

File: pages/styles/protocol.css [hunk 1]
```diff
@@ -125,11 +130,17 @@ main > header h1 {
   line-height: var(--header-height);
   padding-left: 10px;
   display: inline-block;
+  font-weight: 400;
+}
+
+/* When narrow, hide page title to avoid wrapping h1 + search */
+@media only screen and (max-width:810px) {
```
Comment (TimvdLippe) [inline pages/styles/protocol.css]: I had to tweak this to 825 to make it work on Mac.

File: pages/styles/protocol.css [hunk 2]
```diff
@@ -35,6 +35,11 @@ aside {
   border-right: var(--separation-border);
 }
 
+/* hide sidenav on mobile */
```
Comment (TimvdLippe) [inline pages/styles/protocol.css]: I am not sure about this, as there is no way to return to home now. Should we use a hamburger-style menu?

File: pages/styles/protocol.css (hunk 1 above)
Comment (paulirish) [reply pages/styles/protocol.css]: Interesting. I'm on mac, too. we probably got different fonts :)

File: pages/styles/protocol.css (hunk 2 above)
Comment (paulirish) [reply pages/styles/protocol.css]: > I am not sure about this, 

Yah admittedly this is the quick fix to make the site baseline usable on mobile. (it's reallllly bad right now)

> Should we use a hamburger-style menu?

that works for me. can we do as a followup?

**Pull request merged**